src/d3a/models/area/__init__.py

```diff
@@ -350,8 +351,11 @@ def cycle_markets(self, _trigger_event=True, _market_cycle=False, deactivate=Fal
             self._update_descendants_strategy_prices()
             self.should_update_child_strategies = False
 
+        # TODO: Refactor and port the future, spot, settlement and balancing market creation to
+        # AreaMarkets class, in order to create all necessary markets with one call.
+
```

Comment:
Nit: do we already have a ticket for this? Or should we create one?

Reply:
I think we had one, but it is probably buried in Outdated. Creating one quickly